core-web/libs/sdk/client/src/lib/client/content/methods/get-collection/get-collection.ts

```diff
@@ -0,0 +1,285 @@
+     * @memberof GetCollection
+     */
+    private get sort() {
+        return this.#sortBy?.map((sort) => `${sort.field} ${sort.order}`).join(',');
+    }
+
+    /**
+     * This method returns the offset of the content to fetch
+     *
+     * @readonly
+     * @private
+     * @memberof GetCollection
+     */
+    private get offset() {
+        // This could end in an empty response
+        return this.#limit * (this.#page - 1);
+    }
+
+    /**
+     * This method returns the url to fetch the content
+     *
+     * @readonly
+     * @private
+     * @memberof GetCollection
+     */
+    private get url() {
+        return `${this.serverUrl}${CONTENT_API_URL}`;
+    }
+
+    /**
+     * This method returns the current query built
+     *
+     * @readonly
+     * @private
+     * @memberof GetCollection
+     */
+    private get currentQuery() {
+        return this.#query ?? this.#defaultQuery;
+    }
+
+    constructor(
+        requestOptions: Omit<RequestInit, 'body' | 'method'>,
+        serverUrl: string,
+        contentType: string
+    ) {
+        this.requestOptions = requestOptions;
+        this.serverUrl = serverUrl;
+
+        this.#contentType = contentType;
+
+        // We need to build the default query with the contentType field
+        this.#defaultQuery = new QueryBuilder().field('contentType').equals(this.#contentType);
+    }
+
+    /**
+     * This method allows to filter the content by language.
+     *
+     * @param {number} language
+     * @return {*}  {this}
+     * @memberof GetCollection
+     */
+    language(language: number): this {
+        this.#query = this.currentQuery.field('languageId').equals(language.toString());
+
+        return this;
+    }
+
+    /**
+     * This method allows you to fetch the render version of the content
+     *
+     * @param {boolean} render
+     * @return {*}  {this}
+     * @memberof GetCollection
+     */
+    render(render: boolean): this {
+        this.#render = render;
+
+        return this;
+    }
+
+    /**
+     * This method allows you to sort the content by field
+     *
+     * @param {SortByArray} sortBy
+     * @return {*}  {this}
+     * @memberof GetCollection
+     */
+    sortBy(sortBy: SortByArray): this {
+        this.#sortBy = sortBy;
+
+        return this;
+    }
+
+    /**
+     * This method allows you to limit the number of content to fetch
+     *
+     * @param {number} limit
+     * @return {*}  {this}
+     * @memberof GetCollection
+     */
+    limit(limit: number): this {
+        this.#limit = limit;
+
+        return this;
+    }
+
+    /**
+     * This method allows you to set the page of the content to fetch
+     *
+     * @param {number} page
+     * @return {*}  {this}
+     * @memberof GetCollection
+     */
+    page(page: number): this {
+        this.#page = page;
+
+        return this;
+    }
+
+    /**
+     * This method allows you to set a lucene query to filter the content using the Lucene Query Builder.
```

Comment:
Do you think that we should mention "lucene" here? I preffer not, unless they're using `raw`